src/Database/Element/Column.php

```diff
@@ -63,7 +87,7 @@ class Column
     /**
      * @param string $name name of column
      * @param string $type type of column
-     * @param array $settings - list of settings, available keys: null, default, length, decimals, signed, autoincrement, after, first
```

Comment:
But it is outdated ... and some setting options are missing. If this mr will be merged you can simply write Column::S [tab] and you see all available settings listed in your IDE.